src/thunderbots/software/backend_output/main.cpp

```diff
@@ -50,6 +50,8 @@ int main(int argc, char** argv)
     // We loop at 30Hz so we don't overload the network with too many packets
     ros::Rate tick_rate(TICK_RATE);
 
+    Team team = Team(std::chrono::milliseconds(1000));
```

Comment:
Matt was saying that we need a Team further up so that everything is updated and consistent. @MathewMacDougall what do you think?

Reply:
Yes, but we should also be subscribing to the `backend/friendly_team` topic to update that `Team` object. Otherwise we won't be operating on the latest robot data that we receive from the backend/grSim.